wikimedia/wikipedia-ios
# Switch to Xcode Cloud Tests

**Phabricator:** 
https://phabricator.wikimedia.org/T329850

### Notes
I had difficulty getting CircleCI tests to run after switching the configuration Xcode version to 14.3.0, so I decided to fold that investigation time into getting Xcode Cloud PR tests to run. If we're good with this, I'll add a followup PR (or add onto #4500) updating the documentation.

One annoying thing so far:  each test run against this main Wikipedia app increments the build number, which affects what build number is assigned to our nightly build. So we'll see some build number jumps in TestFlight.  The alternative I can think of is to spin up a separate app (or use our existing Experimental or Staging apps) that PR tests are run against instead, but I don't love that idea.

For 1517ebe - I had trouble getting the project to build with the module verifier on (which the previous project recommended settings commit added), so I switched that setting back to off.

### Test Steps
1. Trigger Xcode Cloud workflow "Run Tests" against this branch manually, confirm it succeeds.
2. Confirm unit tests successfully run locally in simulator

Once this is merged, I can enable that workflow and adjust the CircleCI settings to stop building this project.
